tagstudio/src/qt/modals/tag_search.py

```diff
@@ -139,15 +144,27 @@ def update_tags(self, query: str = ""):
                 # f'padding-left: 4px;'
                 f"font-size: 20px;"
                 f"}}"
-                f"QPushButton::hover"
-                f"{{"
+                f"QPushButton::checked{{"
+                f"border-color:{get_tag_color(ColorType.BORDER, self.lib.get_tag(tag_id).color)};"
+                f"color: {get_tag_color(ColorType.PRIMARY, self.lib.get_tag(tag_id).color)};"
+                f"background: {get_tag_color(ColorType.PRIMARY, self.lib.get_tag(tag_id).color)};"
```

Comment:
nitpick: 
`tag_color = self.lib.get_tag(tag_id).color` on the first line of this `for` loop, and then reuse the value here instead of calling `get_tag()` repeatedly?
```suggestion
                f"border-color:{get_tag_color(ColorType.BORDER, tag_color)};"
                f"color: {get_tag_color(ColorType.PRIMARY, tag_color)};"
                f"background: {get_tag_color(ColorType.PRIMARY, tag_color)};"
```

plus the other occurrences around here.